caesars-palace/blocks/header/header.js

```diff
@@ -1,8 +1,12 @@
 import { getMetadata, decorateIcons } from '../../scripts/lib-franklin.js';
 
 // media query match that indicates mobile/tablet width
-const isDesktop = window.matchMedia('(min-width: 1170px)');
-// const MAX_NAV_ITEMS_DESKTOP = 6;
+// const isDesktop = window.matchMedia('(min-width: 1170px)');
+// const isLargeDesktop = window.matchMedia('(min-width: 1440px)');
+// const MAX_NAV_ITEMS_DESKTOP = 7;
+// const MAX_NAV_ITEMS_LARGE_DESKTOP = 9;
+const smallDesktop = { isDesktop: window.matchMedia('(min-width: 1170px)'), maxItems: 7 };
+const largeDesktop = { isLargeDesktop: window.matchMedia('(min-width: 1440px)'), maxItems: 9 };
```

Comment:
Using matchers like this do not take into account window resizes. They will be used once when the block is added but if any screen size changes occur your logic may be incorrect. Media matchers always need to be used inconjunction with a change event.

Reply:
Modified to add a listener for screen change.